config/vkBasalt.json.in

```diff
@@ -1,9 +1,9 @@
 {
   "file_format_version" : "1.0.0",
   "layer" : {
-    "name": "VK_LAYER_VKBASALT_PostProcess32",
+    "name": "VK_LAYER_VKBASALT_PostProcess",
```

Comment:
This change actually breaks things when the `x86` and `x86_64` targets are installed alongside each other. the layer names must be unique.

Reply:
Yeah, indeed, missed it. But not much problem, we can configure it in a similar way as the filename.